recipes/wrf-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.0b1" %}
+
+package:
+  name: wrf-python
+  version: {{ version }}
+
+source:
+  fn: wrf-python-{{ version }}.tar.gz
+  url: https://github.com/NCAR/wrf-python/archive/{{ version }}.tar.gz
+  sha256: 94f05a9719701959702094eafb84555a996bc4f7d75042fb07bca3c35bda9f31
+
+build:
+  number: 1
+  detect_binary_files_with_prefix: true
+  skip: True  # [win32 or np>=112 or (win and py>=35)]
```

Comment:
Not sure why ATM, but something about this selector is causing problems in feedstock conversion. May require some investigation.

xref: https://travis-ci.org/conda-forge/staged-recipes/builds/193381312#L319-L328